packages/webapp/src/containers/Task/TaskComplete/StepOne.jsx

```diff
@@ -27,6 +29,18 @@ function TaskCompleteStepOne({ history, match, location }) {
     history.back();
   };
 
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(
+      setPersistedPaths([
+        `/tasks/${task_id}/complete_harvest_quantity`,
+        `/tasks/${task_id}/complete`,
+        `/tasks/${task_id}/before_complete`,
+        `/tasks/${task_id}/harvest_uses`,
```

Comment:
Since we have a dedicated flow for harvest tasks,  `/tasks/${task_id}/complete_harvest_quantity` and `/tasks/${task_id}/harvest_uses` aren't necessary, are they?

Also, navigating to this page from `read_only` does `setPersistedPaths` in `onComplete` in `packages/webapp/src/containers/Task/TaskReadOnly/index.jsx`, so I think it should be removed.

Reply:
Yeah I was essentially copying the onComplete from readonly just in case there was some reason they were all added. But I agree it shouldn't be necessary.

Removed!

Reply:
Sorry, I was referring to `setPersistedPaths` in another file in the second half in my comment.

Since paths are set in the component itself, I believe this is redundant now.
![Screenshot 2024-04-03 at 9 47 07 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/5735d1b2-4321-4746-a0c1-e77a81ab44d3)

Reply:
Yes that is the function I was copying to put into this different task creation flow. 

That function runs only when using the read only page and clicking complete. But readonly page is not visited in the task creation flow so this function is not run.

Reply:
Oh I see now.. yah hmm I could look into that.

Reply:
I will try it but my concern is that the data available in readonly may not be persisted.

Reply:
Yep good call seems to work fine... deleting! Thanks!

Reply:
Thank you 😂😂